packages/@aws-cdk/custom-resource-handlers/package.json

```diff
@@ -51,7 +54,8 @@
     "@aws-sdk/client-synthetics": "3.421.0",
     "@aws-sdk/client-ecr": "3.421.0",
     "@aws-sdk/client-s3": "3.421.0",
-    "aws-sdk": "^2.1492.0"
+    "aws-sdk": "^2.1492.0",
```

Comment:
I think we can align this version with a fixed version.

Reply:
NVM, looks like this is the version across the repo: https://github.com/search?q=repo%3Aaws%2Faws-cdk%20%22aws-sdk%22%3A&type=code